tests/catalog/test_glue.py

```diff
@@ -46,6 +47,23 @@
 )
 
 
+@mock_aws
+def test_load_catalog_from_impl() -> None:
+    assert isinstance(
+        load_catalog(
+            "catalog",
+            **{
+                PY_CATALOG_IMPL: "pyiceberg.catalog.glue.GlueCatalog",
```

Comment:
[Optional] I think it would be better to explicitly write the config name "py-catalog-impl" in tests

Reply:
fixed